### PR TITLE
Get inventory

### PR DESCRIPTION
**Pull request by simionescuraluca**

### CODE REVIEW

File: src/main/java/com/vinyl/repository/ProductRepository.java [hunk 1]
```diff
@@ -4,6 +4,9 @@
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface ProductRepository extends CrudRepository<Product, Integer> {
+    List<Product> findAll();
```
Comment (dragosslujeru) [inline src/main/java/com/vinyl/repository/ProductRepository.java:11]: this method is already available I think in the CrudRepository, no need to re-declare it

File: src/main/java/com/vinyl/service/ProductService.java [hunk 2]
```diff
@@ -91,6 +93,20 @@ public void removeProductFromStore(String tokenHash, Integer productId) {
         productRepository.delete(productRepository.findById(productId).get());
     }
 
+    public InventoryListDTO getInventory(String tokenHash){
+        validateAdminToken(tokenHash);
```
Comment (dragosslujeru) [inline src/main/java/com/vinyl/service/ProductService.java]: do a merge with master

File: src/main/java/com/vinyl/service/ProductService.java [hunk 3]
```diff
@@ -91,6 +93,20 @@ public void removeProductFromStore(String tokenHash, Integer productId) {
...
+
+        List<Product> productList = productRepository.findAll();
+        List<InventoryDTO> inventory = new ArrayList<>();
```
Comment (dragosslujeru) [inline src/main/java/com/vinyl/service/ProductService.java]: this is a perfect place for stream api

File: src/main/java/com/vinyl/service/ProductService.java (hunk 2 above)
Comment (simionescuraluca) [reply src/main/java/com/vinyl/service/ProductService.java]: done

File: src/main/java/com/vinyl/repository/ProductRepository.java (hunk 1 above)
Comment (simionescuraluca) [reply src/main/java/com/vinyl/repository/ProductRepository.java:11]: it is, but it returns an Iterable and I preffered a List

File: src/main/java/com/vinyl/service/ProductService.java (hunk 3 above)
Comment (simionescuraluca) [reply src/main/java/com/vinyl/service/ProductService.java]: done

**Pull request merged**